modules/m_whois.c

```diff
@@ -60,15 +60,24 @@ struct Message whois_msgtab = {
 int doing_whois_hook;
 int doing_whois_global_hook;
 int doing_whois_channel_visibility_hook;
+int doing_whois_show_idle_hook;
 
 mapi_clist_av1 whois_clist[] = { &whois_msgtab, NULL };
 mapi_hlist_av1 whois_hlist[] = {
 	{ "doing_whois",			&doing_whois_hook },
 	{ "doing_whois_global",			&doing_whois_global_hook },
 	{ "doing_whois_channel_visibility",	&doing_whois_channel_visibility_hook },
+	{ "doing_whois_show_idle",		&doing_whois_show_idle_hook },
 	{ NULL, NULL }
 };
 
+enum whois_idle_approval
```

Comment:
This is useful, but it only gets used in this file while the magic numbers get used in all the other files?

Reply:
Did you have an idea about which header file would be best? I was thinking `hook.h`.